extensions/tekton/client/src/main/java/io/fabric8/tekton/client/V1beta1APIGroupClient.java

```diff
@@ -49,26 +45,26 @@ public V1beta1APIGroupClient(OkHttpClient httpClient, final Config config) {
 
   @Override
   public MixedOperation<Pipeline, PipelineList, Resource<Pipeline>> pipelines() {
-    return new PipelineOperationsImpl(this.getHttpClient(), this.getConfiguration());
+    return Handlers.getOperation(Pipeline.class, PipelineList.class, this.getHttpClient(), this.getConfiguration());
   }
 
   @Override
   public MixedOperation<PipelineRun, PipelineRunList, Resource<PipelineRun>> pipelineRuns() {
-    return new PipelineRunOperationsImpl(this.getHttpClient(), this.getConfiguration());
+    return Handlers.getOperation(PipelineRun.class, PipelineRunList.class, this.getHttpClient(), this.getConfiguration());
   }
 
   @Override
   public MixedOperation<Task, TaskList, Resource<Task>> tasks() {
-    return new TaskOperationsImpl(this.getHttpClient(), this.getConfiguration());
+    return Handlers.getOperation(Task.class, TaskList.class, this.getHttpClient(), this.getConfiguration());
   }
 
   @Override
   public MixedOperation<TaskRun, TaskRunList, Resource<TaskRun>> taskRuns() {
-    return new TaskRunOperationsImpl(this.getHttpClient(), this.getConfiguration());
+    return Handlers.getOperation(TaskRun.class, TaskRunList.class, this.getHttpClient(), this.getConfiguration());
   }
 
   @Override
   public NonNamespaceOperation<ClusterTask, ClusterTaskList, Resource<ClusterTask>> clusterTasks() {
-    return new ClusterTaskOperationsImpl(this.getHttpClient(), this.getConfiguration());
+    return Handlers.getOperation(ClusterTask.class, ClusterTaskList.class, this.getHttpClient(), this.getConfiguration());
```

Comment:
https://github.com/fabric8io/kubernetes-client/pull/3327#pullrequestreview-715741612

I'm confused now. I assumed this was possible until I saw the code in the linked comment. Now I don't get why in those lines we can follow the same procedure.

Reply:
You could.  ClusterServiceBrokerOperationsImpl will be registered so you can get it through Handlers.  However I was attempting to create the minimal changes at first, which would just be to remove the usage of the generated ones.

Reply:
Captured as #3357